packages/babylon/test/fixtures/experimental/dynamic-import/no-plugin/options.json

```diff
@@ -1,4 +1,4 @@
 {
-  "throws": "Unexpected token, expected \"{\" (1:6)",
+  "throws": "This experimental syntax requires enabling the parser plugin: 'dynamicImport' (1:0)",
```

Comment:
I wouldn't use `this experimental syntax` because it's just a question of time. What about just `this syntax`?

Reply:
When it won't be experimenta anymore it will be enabled by default, so the error won't be thrown.

Reply:
- Yes! good point
- No, imagine you use the Babylon's API, you can choose which plugin you want.

It's good to me since i'm not really sure what we can do in the second case.